<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotorPinDialog</class>
 <widget class="QDialog" name="RotorPinDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>325</width>
     <height>125</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="azelLabel">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Az/El pin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="azelCb">
      <item>
       <property name="text">
        <string>Pin 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 9</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dirLabel">
      <property name="text">
       <string>Direction pin:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="dirCb">
      <item>
       <property name="text">
        <string>Pin 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pin 9</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="ccCb">
      <property name="text">
       <string>Rotate counterclockwise</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RotorPinDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RotorPinDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
